assets/src/components/SurveyModal.js

```diff
@@ -41,6 +41,10 @@ const useStyles = makeStyles((theme) => ({
     right: 0,
     width: '100%',
     height: '100%'
+  },
+  surveyButton: {
+    color: theme.palette.primary.main,
+    background: theme.palette.getContrastText(theme.palette.primary.main),
```

Comment:
Nit-pick: Remove the trailing comma? Standard complains, I don't love trailing commas myself, but I know some people do.

Reply:
Yup, don't know why I left that in there. Fixed.